apps/docs/pages/builderdao-cli/tutorial-prepublish.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Tutorial Pre-publish
```

Comment:
I believe that pages should still begin with the H1 title, or else the page will not have a visible title. This impacts readability. See example:

Without H1: 
<img width="975" alt="Screen Shot 2022-05-05 at 10 13 25 AM" src="https://user-images.githubusercontent.com/2707197/166966771-6bba0249-a52e-4d54-bfa9-d4f088c3fa8b.png">

With H1:
<img width="975" alt="Screen Shot 2022-05-05 at 10 10 20 AM" src="https://user-images.githubusercontent.com/2707197/166966192-ea87b441-cc49-4340-9e9a-5fde297dba10.png">